contracts/core/RebalancingSetToken.sol

```diff
@@ -533,17 +586,62 @@ contract RebalancingSetToken is
                 combinedCurrentUnits.push(uint256(0));
             }
 
-            // Compute and push unit amounts of token in rebalancingSet, push 0 if not in set
+            // Compute and push unit amounts of token in nextSet, push 0 if not in set
             if (isInRebalance) {
-                combinedRebalanceUnits.push(
-                    computeUnits(rebalancingSetUnits[indexRebalance], rebalancingSetNaturalUnit)
+                combinedNextUnits.push(
+                    computeUnits(nextSetUnits[indexRebalance], nextSetNaturalUnit)
                 );
             } else {
-                combinedRebalanceUnits.push(uint256(0));
+                combinedNextUnits.push(uint256(0));
             }
         }
     }
 
+    /**
+     * Calculate the amount of nextSets to issue by using the component amounts in the
+     * vault, unitShares following from this calculation.
+     * @return uint256              Amount of nextSets to issue
+     * @return uint256              New unitShares for the rebalancingSetToken
+     */
+    function getIssueAmount()
```

Comment:
We can name this better. It is currently too general and doesn't convey the work done. Perhaps something like `calculateNextSetIssueQuantity`.